source/Test.h

```diff
@@ -118,6 +118,7 @@ class Test {
 	const std::string &Name() const;
 	Status GetStatus() const;
 	const std::string &StatusText() const;
+	const std::set<std::string> RelevantConditions() const;
```

Comment:
Returning a const is an anti-pattern because it inhibits moves. So remove it :smile: